tests/Integration.Tests/Controllers/CoursesController.Tests.cs

```diff
@@ -0,0 +1,243 @@
+using System.Collections.Generic;
+using System.Linq;
+using GovUk.Education.SearchAndCompare.Api.Controllers;
+using GovUk.Education.SearchAndCompare.Api.DatabaseAccess;
+using GovUk.Education.SearchAndCompare.Domain.Filters;
+using GovUk.Education.SearchAndCompare.Domain.Models;
+using GovUk.Education.SearchAndCompare.Domain.Models.Enums;
+using GovUk.Education.SearchAndCompare.Domain.Models.Joins;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.ViewFeatures;
+using MockQueryable.Moq;
+using Moq;
+using NUnit.Framework;
+using NUnit;
+using GovUk.Education.SearchAndCompare.Api.Tests.Integration.Tests.DatabaseAccess;
+using System.Collections.ObjectModel;
+using Microsoft.EntityFrameworkCore;
+using FluentAssertions;
+using Microsoft.Extensions.Logging;
+
+namespace GovUk.Education.SearchAndCompare.Api.Tests.Integration.Tests.Controllers
+{
+    [TestFixture]
+    public class CoursesControllerTests : CourseDbContextIntegrationBase
+    {
+        CoursesController subject;
```

Comment:
`subject` makes the tests harder to read.

Reply:
disagree if you have a list of name items then what is the subject under test?